homeassistant/components/tessie/models.py

```diff
@@ -8,7 +8,7 @@
 
 
 @dataclass
-class TessieVehicle:
+class TessieData:
```

Comment:
With the new generic in runtime_data, this dataclass is no longer needed.

Instead, just directly type it as:

```py
type TessieConfigEntry = ConfigEntry[list[TessieStateUpdateCoordinator]]
```

And store the coordinators in there.

Reply:
In my very next PR I'm going to be adding a second list to the model, so rather than removing it and adding it back again immediately, is this appropriate if the Tessie data is to have more than 1 key?

Reply:
Aight! Let's keep it in that case :)